### DISCUSSION TITLE
进程守护 能不能搞成一个容器 点一点就能够维护好 ，

### DISCUSSION THREAD

**Discussion by YuXunSTC** [1Panel-dev/1Panel#5938] [Ideas]
### 1Panel 版本

v1.10.11

### 请描述您的需求或者改进建议

进程守护 现在还得用linux 命令，能不能搞成一个容器，放几个按钮 ，就能够让用户把自己的项目跑起来，就是尽可能的不用让使用者参与linux 命令，1panel 操作越简单，越是没有技术门槛，使用人才能越来越多，设置有一天 就像用windos一样，所有的人都能用linux 

### 请描述你建议的实现方案

同上 同上

### 附加信息

同上

**Comment by Mr-Ao-Dragon** [comment]
那直接用compose就行了啊，没必要，compose好写的很

**Comment by zhengkunwang223** [comment]
工具箱-进程守护，是这个吗？
